app/assets/javascripts/govuk_publishing_components/lib/cookie-functions.js

```diff
@@ -74,17 +74,6 @@
     window.GOVUK.setCookie('cookie_policy', JSON.stringify(approvedConsent))
   }
 
-  window.GOVUK.denyAllCookieTypes = function () {
```

Comment:
Is this function used by anything?

Reply:
@DilwoarH No - users can't opt-out of 'essential' cookies so this method is not being used, and cannot be used as it stands unless our cookie policy changes.